lms/static/sass/base/_animations.scss

```diff
@@ -258,6 +258,18 @@
   }
 }
 
+@mixin fade-transition {
```

Comment:
While this file might be old and already misused, we tend to keep this type of partial for just CSS and keyframe-based animations. It looks like you're wanting to just define how an element's visual state transitions. For that I'd recommend:

1. Using Bourbon's Transition mixin utility (http://bourbon.io/docs/#transitions) to handle prefixing (e.g. ``@include transition(all 0.2s ease-out);``)
2. Moving the Bourbon-based mixing reference to where you're calling it (e.g. the course image rule).

You can remove these definitions once that's done. In case you need to use this partial in the future for CSS/Keyframe animations, I'd recommend using Bourbon's ``animation()`` mixin - http://bourbon.io/docs/#animation.